src/pirate/psu.c

```diff
@@ -191,8 +201,8 @@ uint32_t psu_enable(float volts, float current, bool current_limit_override) {
 
 void psu_init(void) {
     psu_vreg_enable(false);
-
     // pin and PWM setup
+#if (BP_VER != 7)

```

Comment:
As above, at minimum, add comment with `//BUGBUG - TODO - ` so indicate the why ... and so it can then be changed to capability-based.